tla-io/src/test/scala/at/forsyte/apalache/tla/imp/TestSanyImporterStandardModules.scala

```diff
@@ -1,22 +1,21 @@
 package at.forsyte.apalache.tla.imp
 
 import at.forsyte.apalache.tla.lir.UntypedPredefs._
+import at.forsyte.apalache.tla.lir._
 import at.forsyte.apalache.tla.lir.convenience.tla._
 import at.forsyte.apalache.tla.lir.oper._
 import at.forsyte.apalache.tla.lir.src.{SourcePosition, SourceRegion}
 import at.forsyte.apalache.tla.lir.values._
-import at.forsyte.apalache.tla.lir._
 import org.junit.runner.RunWith
 import org.scalatestplus.junit.JUnitRunner
 
 import scala.io.Source
 
 /**
- * <p>A collection of tests that check how SanyImporter loads the standard modules.</p>
+ * A collection of tests that check how SanyImporter loads the standard modules.
  *
- * <p>If you run this test in an IDE, and the test fails, add the following line to the VM parameters (don't forget to
- * replace <APALACHE_HOME> with the directory where you checked out the project):
- * -DTLA-Library=<APALACHE_HOME>/src/tla </p>
+ * If you run this test in an IDE and the test fails, set environment variable `APALACHE_HOME` to the root of the
+ * Apalache source tree.
```

Comment:
I would argue that if these tests require the file layout we produce after assembling the package, then they should moved into integration tests: IMO, we should not hack the environment to make things that are testing integration look like unit tests.

Reply:
We have done so before by passing `-DTLA-Library`, no? What has changed?

Reply:
I think this is a sign that these always should have been integration tests :)

Reply:
If I need to know what code paths my test hits internally in order to decide whether it's a unit test or an integration test, something's wrong.

Reply:
To my mind, this is not about looking at what code paths it hits internally, it's about considering whether or not we can test an isolated unit, or whether we are testing the integration of our system with (relatively) external systems. In this case, we are testing the integration with the file system and the external dependency on SANY's importing logic. That seems like a clear case of integration testing to me. What do you think? Maybe we are disagreeing on terms here?

I guess, thinking strictly, if we wanted a unit test for this same logic, we'd stub or mock out SANY's import logic.

Reply:
I agree that this is, conceptually, an integration test. I also believe that SANY is such an important dependency that we should be able to run Scalatest fixtures against it, the same we way currently run unit tests.